src/test/app/Flow_test.cpp

```diff
@@ -747,8 +747,8 @@ struct Flow_test : public beast::unit_test::suite
 
         auto const timeDelta = Env{*this}.closed ()->info ().closeTimeResolution;
 
-        for(auto const& d: {-timeDelta*100, +timeDelta*100}){
-            auto const closeTime = fix1141Time () + d;
+        for(auto const& d: {-100, +100}){
+            auto const closeTime = fix1141Time () + d * timeDelta;
```

Comment:
The warning looks bogus to me, I believe the code is correct as written. However of course we do need to remove the warning. Instead of this, does `auto const& d : {-100*timeDelta, 100*timeDelta}` make the warning go away? If so I'd prefer to remove the warning that way. If other's disagree (either the code is wrong or the current fix is better) I'm happy to be overruled.